ui/raidboss/data/04-sb/raid/o2s.js

```diff
@@ -60,8 +70,10 @@
     },
     {
       id: 'O2S Death\'s Gaze',
-      regex: / 14:236F:Catastrophe starts using/,
-      regexDe: / 14:236F:Katastroph starts using/,
+      regex: / 14:236F:Catastrophe starts using Death'S Gaze/,
```

Comment:
Should the S in `Death's Gaze` be capitalized?

Reply:
Seams python went crazy as i have a rule which should replace 'S with 's but seams it sometimes did it the other way around. Fixed with next commit.